web/vtadmin/src/components/routes/workflow/WorkflowDetails.tsx

```diff
@@ -243,18 +288,22 @@ export const WorkflowDetails = ({ clusterID, keyspace, name }: Props) => {
                     title="Table Copy State"
                 />
             )}
-            <h3 className="mt-8 mb-4">Recent Logs</h3>
-            {streams.map((stream) => (
-                <div className="mt-2" key={stream.key}>
-                    <DataTable
-                        columns={LOG_COLUMNS}
-                        data={orderBy(stream.logs, 'id', 'desc')}
-                        renderRows={renderLogRows}
-                        pageSize={10}
-                        title={stream.key!}
-                    />
-                </div>
-            ))}
+            {streams.length <= 8 && (
```

Comment:
I feel like we should have some sort of else condition here where we display something to indicate that we've elided the otherwise expected UI element / display content due to the larger number of shards?

Reply:
Done.

<img width="1440" alt="Screenshot 2024-09-08 at 2 31 43 PM" src="https://github.com/user-attachments/assets/4bcf8eef-687d-4752-abf6-517c8d39bd21">

(PS: Used `streams.length <= 1` just for the preview)